src/NodaTime/Calendars/WondrousYearMonthDayCalculator.cs

```diff
@@ -0,0 +1,385 @@
+﻿// Copyright 2014 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NodaTime.Calendars.Wondrous;
+using NodaTime.Utility;
+using System;
+
+namespace NodaTime.Calendars
+{
+    /// <summary>
+    ///     See <see cref="CalendarSystem.GetWondrousCalendar" /> for details.
+    /// </summary>
+    internal sealed class WondrousYearMonthDayCalculator : YearMonthDayCalculator
+    {
+        private const int WondrousMinYear = 1;
+        private const int WondrousMaxYear = 1000;
+
+        private const int AverageDaysPer10Years = 3652; // Ideally 365.2425 per year...
+        private const int DaysInMonth = 19;
+        private const int MonthsInYear = 19;
+
+        private const int Month18 = 18;
+        private const int Month19 = 19;
+
+        public const int AyyamiHaMonth0 = WondrousCalendarHelper.AyyamiHaMonth0;
+
+        private const int UnixEpochDayAtStartOfYear1 = -45941;
+
+        internal WondrousYearMonthDayCalculator()
+            : base(WondrousMinYear,
+                WondrousMaxYear - 1,
+                AverageDaysPer10Years,
+                UnixEpochDayAtStartOfYear1)
+        {
+            ConsoleWriteLine("~A Constructed");
```

Comment:
Please remove both lines.

Reply:
All the ConsoleWriteLine calls can be removed. I was using that before I was able to get a visual test runner to work.